kairos-contracts/demo-contract-tests/tests/integration_tests.rs

```diff
@@ -0,0 +1,64 @@
+mod test_fixture;
+#[cfg(test)]
+mod tests {
+    use crate::test_fixture::TestContext;
+    use casper_types::{account::AccountHash, U512};
+
+    #[test]
+    fn should_install_deposit_contract() {
+        let mut fixture: TestContext = TestContext::new();
+        fixture.install_demo_contract(fixture.account_1);
+    }
+
+    fn setup() -> (TestContext, AccountHash, AccountHash) {
+        let fixture: TestContext = TestContext::new();
+        let installer = fixture.account_1;
+        let user = fixture.account_2;
+        (fixture, installer, user)
+    }
```

Comment:
If `account_1` is not capturing the semantics for the test case correctly, then why not just rename the field from `account_1` to `installer`.

Reply:
Because any account can be an installer. If I want to use the account_x as the installer when writing tests then I can assign it as such:

let installer = fixture.account_x